src/ci/docker/scripts/musl-toolchain.sh

```diff
@@ -0,0 +1,64 @@
+set -ex
```

Comment:
Could you add some comments to the top of this file indicating sort of what it's doing? 

For example describing a bit about what `musl-cross-make` is and how we're using it to build a toolchain (and how that affects a version of musl etc).

Nothing too too long, just probably good hygiene practice!